client/app/components/faq/faqdetails.js

```diff
@@ -3,19 +3,17 @@ import {
 	Text,
 	View,
 } from 'react-native';
-import { FaqStyles } from '../../styles/faqstyles'
-import { Header } from '../header'
-import { ListSlider } from '../listslider'
+import FaqStyles from '../../styles/FaqStyles'
+import Header from '../Header'
+import ListSlider from '../ListSlider'
 
 class FaqDetails extends React.Component {
 	static navigationOptions = ({ navigation }) => ({
-		header: (
-			<Header
-				title="Information"
-				hasBackButton
-				navigation={navigation}
-			/>
-		)
+		header: <Header
```

Comment:
What do you think about keep vs removing the parenthesis here?

Reply:
Add parenthesis around it?  I think we should add them in.  Not sure why it was removed.  Thanks for the catch!